src/client/components/boardColumn/index.js

```diff
@@ -54,6 +60,40 @@ class List extends React.Component {
     this.setState({isEditing: false});
   }
 
+  editColumn() {
+    let that = this;
+
+    that.setState({
+      isColumnTitleEditing: !this.state.isColumnTitleEditing,
+      allowDrag: this.state.isColumnTitleEditing,
+    });
+  }
+
+  generateColumnTitle() {
+    if (this.state.isColumnTitleEditing) {
+      return (
+        <section className="column-title-editing">
+          <input type="text"
+            ref={(c) => {
```

Comment:
what is the purpose of this?

Reply:
to get the input so I can use this.input.focus() to automatically set focus to input after click edit